downstream/modules/platform/ref-controller-settings-scheduling-jobs.adoc

```diff
@@ -0,0 +1,15 @@
+[id="ref-controller-settings-scheduling-jobs"]
+
+= Settings for scheduling jobs
+
+The task manager periodically collects tasks that need to be scheduled and determines what instances have capacity and are eligible for running them. The task manager has the following workflow:
+
+. Find and assign the control and execution instances.
+. Update the job's status to waiting.
+. Message the control node through `pg_notify` for the dispatcher to pick up the task and start running it.
+
+If the scheduling task is not completed within `TASK_MANAGER_TIMEOUT` seconds (default 300 seconds), the task is terminated early. Timeout issues generally arise when there are thousands of pending jobs.
+
+One way the task manager limits how much work it will do in a single run is the setting `START_TASK_LIMIT`. This limits how many jobs it will start in a single run. The default is 100 jobs. If more jobs are pending, a new scheduler task is scheduled to run immediately after. Users who are willing to have potentially longer latency between when a job is launched and when it starts, in order to have greater overall throughput, may consider increasing the `START_TASK_LIMIT`. To see how long individual runs of the task manager take, use the Prometheus metric `task_manager__schedule_seconds`, available in `/api/v2/metrics`.
```

Comment:
it will do -> it can do

is the START_TASK_LIMIT setting.

in order to have -> to have

may consider -> can consider